tenets/codelingo/code-review-comments/declare-empty-slice/codelingo.yaml

```diff
@@ -4,7 +4,9 @@ funcs:
     body: |
       function(sliceType) {
         sliceType = sliceType.split(' ').join('');
-        return  sliceType.substring(0, sliceType.indexOf("{"))
+        if (sliceType.substring(sliceType.length - 4, sliceType.length) == "{}{}" && sliceType.indexOf("interface") !== 0)
```

Comment:
Can you please add a comment explaining this exception? I couldn't quite understand it. Like, why `sliceType.length - 4`, for example.

Reply:
If it has to do with the fact the `{}` in `interface{}` is part of the type, the same thing applies for `struct{}`.

Reply:
I am checking for empty interface, can you please give me an example of struct and I will include that too